docs/deploy.md

```diff
@@ -79,7 +79,12 @@ Run through them again for a second cluster to use with the extended example sho
    helm install gateway-api-controller \
       oci://public.ecr.aws/aws-application-networking-k8s/aws-gateway-controller-chart\
       --version=v0.0.12 \
-      --set=aws.region=$AWS_REGION --set=serviceAccount.create=false --namespace aws-application-networking-system
+      --set=serviceAccount.create=false --namespace aws-application-networking-system \
+      # Region, clusterVpcId, awsAccountId are required for fargate use case
```

Comment:
can you replace "fargate use case" to  something " for the case where IMDS are not available for discovering Region, clusterVpcId, awsAccountId,  for example  fargate usecase"